Makefile

```diff
@@ -3,7 +3,10 @@ WEBPACK := yarn build-acceptance
 
 bootstrap: develop init-config run-dependent-services create-db apply-migrations build-platform-assets
 
-develop: ensure-pinned-pip setup-git install-js-dev install-py-dev
+develop: ensure-venv upgrade-pip
```

Comment:
Once you move ensure-venv to inside do.sh setup-git, you can remove both these targets.